gtsfm/scene_optimizer.py

```diff
@@ -87,6 +99,42 @@ def __init__(self, matcher: MatcherBase, verifier: VerifierBase):
         self.matcher = matcher
         self.verifier = verifier
 
+    def __compute_metrics(
+        self,
+        i2Ri1_computed: Optional[Rot3],
+        i2Ui1_computed: Optional[Rot3],
+        i2Ri1_expected: Rot3,
+        i2Ui1_expected: Unit3,
+    ) -> Tuple[float, float]:
+        """[summary]
+
+        Args:
+            i2Ri1_computed (Optional[Rot3]): [description]
+            i2Ui1_computed (Optional[Rot3]): [description]
+            i2Ri1_expected (Rot3): [description]
+            i2Ui1_expected (Unit3): [description]
+
+        Returns:
+            Tuple[float, float]: [description]
+        """
+
+        R_error = comp_utils.compute_relative_rotation_angle(
+            i2Ri1_computed, i2Ri1_expected
+        )
+
+        U_error = comp_utils.compute_relative_unit_translation_angle(
+            i2Ui1_computed, i2Ui1_expected
+        )
+
+        logging.debug(
```

Comment:
might get too verbose to print for every pair, maybe median and average are more useful?

Reply:
(but still dump all to JSON)

Reply:
Median and average won't help here. I think maybe bin the errors and dump the histogram?

I want to decide on the format/tech used to dump and follow up in a second pair.

Reply:
kicking for later